src/store/lib/deck-grouping.ts

```diff
@@ -75,6 +75,15 @@ function countGroup(cards: Card[], quantities?: Record<string, number>) {
   return cards.reduce((acc, card) => acc + (quantities?.[card.code] ?? 0), 0);
 }
 
+function countXPGroup(cards: Card[], quantities?: Record<string, number>) {
+  return cards.reduce(
+    (acc, card) =>
+      acc +
+      (quantities?.[card.code] ?? 0) * ((card.taboo_xp || 0) + (card.xp || 0)),
```

Comment:
We have a `countExperience` helper for getting card experience, which also takes into account exceptional and other such things. We should use it here:

```ts
function countXPGroup(cards: Card[], quantities?: Record<string, number>) {
  return cards.reduce(
    (acc, card) => acc + countExperience(card, quantities?.[card.code] ?? 0),
    0,
  );
}
```